packages/workflow/src/records/audit.ts

```diff
@@ -13,25 +13,29 @@ import { getEventType } from '@workflow/features/registration/utils'
 import { ValidRecord } from '@opencrvs/commons/types'
 import { METRICS_URL } from '@workflow/constants'
 
-type AuditAction = 'in-progress-declaration' | 'new-declaration'
+type AuditAction =
+  | 'in-progress-declaration'
+  | 'new-declaration'
+  | 'marked-as-duplicate'
 
 export async function auditEvent(
   action: AuditAction,
   bundle: ValidRecord,
   authToken: string
 ) {
   const eventType = getEventType(bundle).toLowerCase()
-  const res = await fetch(
-    new URL(`/events/${eventType}/${action}`, METRICS_URL).href,
-    {
-      method: 'POST',
-      body: JSON.stringify(bundle),
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: `Bearer ${authToken}`
-      }
+  const url =
+    action !== 'marked-as-duplicate'
+      ? new URL(`/events/${eventType}/${action}`, METRICS_URL).href
+      : new URL(`/events/${action}`, METRICS_URL).href
+  const res = await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(bundle),
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${authToken}`
```

Comment:
We might as well change the `marked-as-duplicate` endpoint route in metrics instead of complicating things here